pkg/sources/git/git.go

```diff
@@ -979,19 +986,20 @@ func resolveHash(repo *git.Repository, ref string) (string, error) {
 	return resolved.String(), nil
 }
 
-func stripPassword(u string) (string, error) {
+func stripPassword(u string) (string, string, error) {
```

Comment:
**Nit:** Please add a doc comment here. Now that the function is returning two strings, it would be good to note which is which, and why we would want to return the password.